models/article-models.js

```diff
@@ -16,4 +16,19 @@ exports.selectArticleById = (article_id) => {
     });
 };
 
-
+exports.selectAllArticles = () => {
+  return db
+    .query(
+      `
+  SELECT articles.author, articles.title, articles.topic, articles.created_at, articles.votes, articles.article_img_url, 
+  COUNT(comments.comment_id) AS comment_count
```

Comment:
You could use casting here to make the comment count a number rather than a string